### DISCUSSION TITLE
Chinese text in repo names normal?

### DISCUSSION THREAD

**Discussion by jonnyhotchkiss** [LSPosed/LSPosed#2888] [General]
First tab lets you search for modules,  but many aren't found. What are the sources,  and are they configurable?

Also is the Chinese text normal ?!
I previously read it could relate to incorrect display (of characters) but after searching repo on gh for related issues and discussions I noticed the changelog is bilingual, so not suspecting a display bug

**Comment by JasonKhew96** [comment]
> First tab lets you search for modules, but many aren't found. What are the sources, and are they configurable?

https://github.com/Xposed-Modules-Repo and it's not configurable

> Also is the Chinese text normal ?!
I previously read it could relate to incorrect display (of characters) but after searching repo on gh for related issues and discussions I noticed the changelog is bilingual, so not suspecting a display bug

yes, our Chinese userbase is very large, and most of our developers are Chinese speakers.
